File: Funcional/ATV_1_LISTA_1/exercicio10.hs
mediaAluno n1 n2
    | media >= 7 = "Aprovado, nota: " ++ show media
    | media >= 3 = "Exame, nota: " ++ show media
    | otherwise = "Reprovado, nota: " ++ show media
    where media = (n1 + n2) / 2

main = do
    print (mediaAluno 7 7)
    print (mediaAluno 3 3)
    print (mediaAluno 2 2)

File: Funcional/ATV_1_LISTA_1/src/exercicio10.py
def mediaAluno(n1, n2):
    media = (n1 + n2) / 2
    if media >= 7:
        return "Aprovado, nota: {}".format(media)
    elif media >= 3:
        return "Exame, nota: {}".format(media)
    return "Reprovado, nota: {}".format(media)

print(mediaAluno(5, 6))
